upup/models/cloudup/resources/addons/networking.weave/v1.9.4.yaml

```diff
@@ -0,0 +1,118 @@
+kind: ClusterRole
```

Comment:
How does this work with systems w/o rbac setup? How does this work with 1.5 systems? 

I am guessing that we need to check for rbac and check for k8s version.

Thoughts?